pkg/apis/jaegertracing/v1/jaeger_types.go

```diff
@@ -183,6 +183,9 @@ type JaegerCommonSpec struct {
 
 	// +optional
 	ServiceAccount string `json:"serviceAccount,omitempty"`
+
+	// +optional
+	HostNetWork *bool `json:"hostNetWork,omitempty"`
```

Comment:
@jpkrohling If this is only applicable to agent (currently), shouldn't it only be added to the `JaegerAgentSpec`?

Reply:
Currently I'm only using it on the agent, but I'm not sure if that will change in the future. If you expect to only add to the JaegerAgentSpec, I can change that too.

Reply:
If you could add to the agent spec for now - and if a requirement arises in the future, we can move it to the common spec.

Reply:
ok